### PR TITLE
chore: August dependency bump 🌲  & 🧹

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "locate",
   "version": "2.4.10",
   "dependencies": {
-    "agrc": "agrc-widgets#~6.0",
+    "agrc": "agrc-widgets#6.1.6",
```
Comment (steveoh) [inline bower.json:5]: should this be agrc.widgets?

Comment (stdavis) [reply bower.json:5]: I think so since it's an old dojo project. Or am I missing something?

**Comment by stdavis** [comment]
Oh, now I see the build error. Strange, since it installed on my local machine without issues...

File: bower.json (hunk 1 above)
Comment (stdavis) [reply bower.json:5]: Oh, now I see the build error. Strange, since it installed on my local machine without issues...

**Review by steveoh — APPROVED** [review]

**Pull request merged**